### PR TITLE
Update to python3.9 and ansible-builder 3.0.0; merge repos

### PR DESCRIPTION
**Pull request by Venator-Fox**
Hello, this is a proposal to use just one EE, managed by OIT SE but with shared maintainers. If this is allowed, please review this PR so other groups (ie. the ITCP CS group and OIT RC) can continue to use your EE builds.

Summary of these commits:
- Fixes current build issues
- Merges modules and collections from CS/RC-Dev (ie the crowdstrike falcon modules)
- Merges execution-environment.yml and update label from v1 to v3
- Switch from stream8 to stream9 base
- Updates from python3.8 to 3.9 for builder requirements
- Updates ansible-builder from 1.2.0 to 3.0.0

### CODE REVIEW

**Review by davegoldhammer — COMMENTED** [review]
@Venator-Fox thank you for this PR! Your changes look good from my perspective, but I added @rchap4 as a reviewer, as he is much more python savvy than I am.

I do like the idea of collaborating and sharing this repository, but I'm not sure if there are any potential pitfalls. I suppose we always have the flexibility to point at specific builds if an issue comes up that impacts one of the teams. I know we've had a few little anomalies where we have had to implement a workaround.

**Comment by Venator-Fox** [comment]
> @Venator-Fox thank you for this PR! Your changes look good from my perspective, but I added @rchap4 as a reviewer, as he is much more python savvy than I am.
> 
> I do like the idea of collaborating and sharing this repository, but I'm not sure if there are any potential pitfalls. I suppose we always have the flexibility to point at specific builds if an issue comes up that impacts one of the teams. I know we've had a few little anomalies where we have had to implement a workaround.

Thanks @davegoldhammer, I am hoping to give the collaborating thing "a shot"; which hopefully should not be an issue esp as long as the latest tag is not always used on pulls. As for the anomalies I assume? those are pinning older versions. Those were left untouched in the [requirements.txt ](https://github.com/UCBoulder/cs-awx-ee/blob/cs-fork-from-sepe/spec/requirements.txt#L1-L10)

I guess if we start finding that each other's requirements break one another workflows; then continue using forks.

File: README.md [hunk 1]
```diff
@@ -11,24 +11,24 @@ Originally cloned from the [Ansible awx-ee repository](https://github.com/ansibl
...
 bash prompt:
 
 ```
-$ podman run -it 9797da5f7526 /bin/bash
-bash-4.4# python3 -V
-Python 3.8.13
+$ podman run -it f6c5989cd851 /bin/bash
+bash-5.1# python3 -V
+Python 3.9.19
 bash-4.4# exit
```
Comment (rchap4) [inline README.md]: @Venator-Fox , Can we update the bash version shown in this example to match the shell prompt in line 30?

**Review by rchap4 — APPROVED** [review]
Looks good @Venator-Fox, thanks for these contributions.  Just one comment on README.md to keep the documentation tidy.  However, I won't let that hold back the approval.  Feel free to commit any changes there and we'll proceed with accepting this PR.
-Rory

File: README.md (hunk 1 above)
Comment (Venator-Fox) [reply README.md]: Thanks for catching that; I have fixed that typo in the readme.

**Comment by Venator-Fox** [comment]
@willhaines and @davegoldhammer I believe this PR is also able to resolve #27 as well as possibly allow #29 and #30 to be completed more easily now.

**Pull request merged**